acoular/spectra.py

```diff
@@ -176,12 +176,7 @@ class PowerSpectra(BaseSpectra):
 
     #: The :class:`~acoular.calib.Calib` object that provides the calibration data,
     #: defaults to no calibration, i.e. the raw time data is used.
-    #: **deprecated, will be removed in version 25.01**: use
-    #: :attr:`~acoular.sources.TimeSamples.calib` property of :class:`~acoular.sources.TimeSamples`
-    #: objects
-    calib = Property(desc='calibration object (deprecated, will be removed in version 25.01)')
-
-    _calib = Instance(Calib)
+    calib = Instance(Calib)
```

Comment:
calib capabilities should be removed from PowerSpectra (requires changes in `calc_csm` method)